Demo/ViewController.swift

```diff
@@ -45,15 +45,17 @@ class ViewController: UITableViewController {
         case paymentDetails
         case listTransferMethod
         case addTransferMethod
-        case receipts
+        case userReceipts
+        case prepaidCardReceipts
         case transferFunds
```

Comment:
Could remove `static let count = 5` from enum Example? 

Also, replace `enum Example: Int {` with `enum Example: Int, CaseIterable {`. 

And replace `tableView(_: numberOfRowsInSection)` with 
```
override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
    return Example.allCases.count
}
```

Reply:
Good idea, will do